tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml

```diff
@@ -59,8 +59,7 @@ jobs:
          python3 -m pip uninstall -y ort-nightly-gpu ort-nightly onnxruntime onnxruntime-gpu onnxruntime-training onnxruntime-directml ort-nightly-directml onnx -qq
          cp $(Build.SourcesDirectory)/tools/ci_build/github/linux/docker/scripts/manylinux/requirements.txt $(Build.BinariesDirectory)/requirements.txt
          # Test ORT with the latest ONNX release.
-         export ONNX_VERSION=$(cat $(Build.SourcesDirectory)/cmake/external/onnx/VERSION_NUMBER)
-         sed -i "s/git+http:\/\/github\.com\/onnx\/onnx.*/onnx==$ONNX_VERSION/" $(Build.BinariesDirectory)/requirements.txt
+         sed -i "s/git+http:\/\/github\.com\/onnx\/onnx.*/onnx/" $(Build.BinariesDirectory)/requirements.txt
```

Comment:
You make it to auto-fetch the latest ONNX release, then whenever ONNX publishes a new minor release, this pipeline will  fail. Because as what you are doing now, a lot of work needs to be done.

Reply:
With the code the old way it would fail whenever we tried to test a pre-release version of ONNX (because `ONNX_VERSION` would be set to something like `1.12.0rc3`), and with the code the new way it seems to match the intent of the comment much better.
But I'm happy to do whatever you want. Do you want me to revert this change?

Reply:
I think a fixed number is better. Maybe there are better solutions, but putting "==1.12" there would be the easiest.